proto/slinky/marketmap/v1/query.proto

```diff
@@ -47,6 +47,9 @@ message GetMarketMapResponse {
   // Version is the schema version for the MarketMap data structure and query
   // response.
   uint64 version = 3;
+
+  // ChainId is the chain identifier for the market map.
+  string chain_id = 4;
```

Comment:
Adding a ChainID to the response so that we can uniquely associate marketmaps with chains.